support setting UUID with path param (#1)
AllanChain/grafana-feishu: README.md

The program needs two environment variables:

> - `FEISHU_WEBHOOK_BASE`: (Optional) The web hook base URL to push Feishu notifications. Default is `https://open.feishu.cn/open-apis/bot/v2/hook`

Reviewer: Well, I don't think there is a need to configure `FEISHU_WEBHOOK_BASE`. What else can it be?

We can remove `FEISHU_WEBHOOK_BASE` and only document `FEISHU_WEBHOOK` environment variable.

```suggestion
- `FEISHU_WEBHOOK`: (Optional) The web hook URL to push Feishu notifications. Should be something like `https://open.feishu.cn/open-apis/bot/v2/hook/a21843-123-123-abc987`. If it is not provided, the bot UUID should be provided in the path. See below for detail.
```

Author: I thought there maybe corner cases where people might want to define there own API base. I don't know if lark suite (the oversea version) has the same bot functionalities but with 'larksuite.com' domain name. Or feishu might decide to roll out v3 API instead of v2. 

To minimize burden on users, we can remove the document. But maybe we want to keep the code configurable, as well as the prebuit container images.

Anyway, you decide this.

    container_name: grafana-feishu
    restart: always
    environment:

>       - FEISHU_WEBHOOK_BASE=${FEISHU_WEBHOOK_BASE}

Reviewer: Therefore, keep this the same as before:

```suggestion
      - FEISHU_WEBHOOK=${FEISHU_WEBHOOK}
```

      - WEBHOOK_AUTH=${WEBHOOK_AUTH}
```

The exposed port is `2387`.

After setting up the server, go to Grafana > "Alerting" > "Contact points", add a new contact point with integration as "Webhook". Fill in the URL and credentials. 

> The URL should be like `http://grafana-feishu:2387/{botUUID}`, where `{botUUID}` is the UUID of the bot you created in Feishu, i.e. the last part of the bot webhook URL.
> 
> In the previous version, a whole bot URL (including the UUID) can be set with the environment variable `FEISHU_WEBHOOK`, and the bot UUID is not needed in Grafana. This is still supported.

Reviewer: Maybe more clear and less verbose.

```suggestion
If `FEISHU_WEBHOOK` is provided, the URL should be `http://grafana-feishu:2387`. If not, the URL should be `http://grafana-feishu:2387/{botUUID}`, where `{botUUID}` is the UUID of the bot you created in Feishu, i.e. the last part of the bot webhook URL.
```
